--- blackbox-0.62.1/nls/Makefile.am	Fri Jan 25 10:17:45 2002
+++ blackbox-0.62.1.with_pipe/nls/Makefile.am	Fri Mar  8 21:25:26 2002
@@ -1,7 +1,7 @@
 # nls/Makefile.am for Blackbox - an X11 Window Manager
 
 NLSTEST = @NLS@
-SUBDIRS = C da_DK de_DE es_ES et_EE fr_FR it_IT ja_JP nl_NL pt_BR ru_RU \
+SUBDIRS = C da_DK de_DE es_ES et_EE fr_FR it_IT ja_JP nl_NL pl_PL pt_BR ru_RU \
 	sl_SI sv_SE tr_TR zh_CN
 MAINTAINERCLEANFILES = Makefile.in blackbox-nls.hh
 DISTCLEANFILES = blackbox-nls.hh
--- blackbox-0.62.1/configure.in	Fri Jan 25 11:50:18 2002
+++ blackbox-0.62.1.with_pipe/configure.in	Fri Mar  8 21:51:15 2002
@@ -243,6 +243,7 @@
 nls/it_IT/Makefile
 nls/ja_JP/Makefile
 nls/nl_NL/Makefile
+nls/pl_PL/Makefile
 nls/pt_BR/Makefile
 nls/ru_RU/Makefile
 nls/sl_SI/Makefile
