--- blackbox-0.65.0/data/Makefile.am.orig	Fri Nov  1 17:53:06 2002
+++ blackbox-0.65.0/data/Makefile.am	Fri Nov  1 17:53:32 2002
@@ -14,7 +14,7 @@
 
 install-data-local: menu
 	test -f $(DESTDIR)$(pkgdatadir)/menu || \
-	$(INSTALL_DATA) menu $(DESTDIR)$(pkgdatadir)
+	$(INSTALL_DATA) menu $(DESTDIR)$(sysconfdir)
 
 uninstall-local:
 	rm -f $(pkgdatadir)/menu
--- blackbox-0.65.0/src/Makefile.am.orig	Fri Nov  1 17:54:34 2002
+++ blackbox-0.65.0/src/Makefile.am	Fri Nov  1 17:56:32 2002
@@ -19,8 +19,8 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER 
 # DEALINGS IN THE SOFTWARE.
 
-DEFAULT_MENU=$(pkgdatadir)/menu
-DEFAULT_STYLE=$(pkgdatadir)/styles/Results
+DEFAULT_MENU=$(sysconfdir)/menu
+DEFAULT_STYLE=$(sysconfdir)/styles/Results
 
 CPPFLAGS= @CPPFLAGS@ @SHAPE@ @ORDEREDPSEUDO@ \
 @DEBUG@ @NLS@ @TIMEDCACHE@ \
